scss/components/_switch.scss

```diff
@@ -208,6 +208,10 @@ $switch-paddle-transition: all 0.25s ease-out !default;
   // <input> element
   .switch-input {
     @include switch-input;
+      // indicate a disabled switch
+    &:disabled ~ label {
```

Comment:
I don't like this selector (non BEM-like) but this is the CSS architecture of the component, so good enough.